src/docs/rules/handleCallbackErrRule.md

```diff
@@ -32,6 +32,14 @@ regexp pattern.
   name matches any string that contains `err` or `Err` (for example, `err`, `error`,
   `anyError`, `some_err` will match).
 
+In addition to the string we may specify an options object with the following property:
+
+- `allowProperties`: (`true` by default) When this is set to `false` the rule will not
+  report unhandled errors as long as the error object is handled without accessing any of its
+  properties at least once. For instance, `(err) => console.log(err.stack)` would report an
+  issue when `allowProperties` is set to `false` because `err` is not handled on its
+  own.
+
```

Comment:
Done, I think that's it then. I'll merge and do a release sometime tonight. Thanks for your help.